src/navigator/types.tsx

```diff
@@ -167,7 +167,6 @@ export type StackParamList = {
   }
   [Screens.Simplex]: {
     simplexQuote: SimplexQuote
-    userIpAddress: string
```

Comment:
Could this and related SimplexScreen changes be in a separate PR?

Reply:
I'd rather keep it because it makes the simplex onPress function cleaner.